### ISSUE TITLE
Possible to keep pie chart slice sequence based on data provided?

### ISSUE THREAD

**Issue by jolewis-ddn** [naver/billboard.js#1648]
## Description
It looks like pie charts are displayed in order from largest slice to smallest slice. Is it possible to keep the sequence static - i.e. matching the order provided?

## Steps to check or reproduce

```
<span id='chart' class='miniJSRChart'></span>
<script>
 var chart = bb.generate({
 bindto: "#chart",
 x: "x",
 size: { height: 250, width: 250 },
 title: { text: 'Pie' },
 data: {
 type: "pie",
 columns: [ ['a', 1], ['b', 2], ['c', 0], ['d', 0 ] ],
 labels: { colors: { 'a': 'black', 'b': 'white', 'c': 'black', 'd': 'black' } }
 },
 pie: { label: { format: function(value, ratio, id) { return value; }}},
color: { pattern: [ 'SeaShell', 'MediumSeaGreen', 'CornflowerBlue', 'Pink' ] },
 });
</script>
```

![image](https://user-images.githubusercontent.com/31732476/91732693-92d01480-eb76-11ea-9ec3-90019502cf73.png)

Question: Is it possible to display the "a" value (1) first (i.e. in the top-right area - from "12 to 4pm") instead of last (i.e. in the top-left area - from "8pm to 12")

**Comment by michkami** [comment]
You can sort your columns by adding `data: {order : "asc"}` to sort your columns.
It was fixed by `1.9.0` release (https://github.com/naver/billboard.js/issues/34#issuecomment-506274826)

**Comment by jolewis-ddn** [comment]
Perfect! Thanks @michkami
